guidelines/enhancement_template.md

```diff
@@ -74,6 +75,7 @@ around the enhancement process.
 ## Release Signoff Checklist
 
 - [ ] Enhancement is `implementable`
```

Comment:
@dhellmann feels like we want to change the above line a bit to reflect what implementable is to us - any suggestions? maybe something like: 

`have agreement from all stakeholders and enhancement will be implemented in the near future`

Reply:
If we're going to remove this section (see my earlier comment) we would want to say that somewhere else. Maybe in the instructions around line 26 or 28?

Reply:
make sense to me!